src/otx/v2/adapters/torch/mmengine/mmdet/modules/losses/cross_focal_loss.py

```diff
@@ -0,0 +1,180 @@
+"""Cross Focal Loss for ignore labels."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from __future__ import annotations
+
+import torch
+from mmdet.models.losses.focal_loss import py_sigmoid_focal_loss, sigmoid_focal_loss
+from mmdet.models.losses.varifocal_loss import varifocal_loss
+from mmdet.registry import MODELS
+from torch import nn
+from torch.nn import functional
+
+# pylint: disable=too-many-arguments, too-many-locals, too-many-instance-attributes, unused-argument
+
+
+def cross_sigmoid_focal_loss(
+    inputs: torch.Tensor,
+    targets: torch.Tensor,
+    weight: torch.Tensor | None = None,
+    num_classes: int | None = None,
+    alpha: float = 0.25,
+    gamma: float = 2.0,
+    reduction: str = "mean",
+    avg_factor: int | torch.Tensor | None = None,
+    use_vfl: bool = False,
```

Comment:
What is `vfl`? Varifocal loss? I would use a more descriptive name. It may not be clear to someone who is new to the content.